### DISCUSSION TITLE
Splitting up the application

### DISCUSSION THREAD

**Discussion by shivan-s** [mrdbourke/nutrify#27] [Ideas]
Do you think it will be worth creating an "organisation" for nutrify? Then, separating the repository: one repo for the web application, one for EDA and training the CV model?

Rather than having one repo for everything. This can make it a lot cleaner.

**Comment by mrdbourke** [comment]
Yes, that's a good idea

That'd make it something like github.com/nutrify/nutrify right?

The single repo will eventually get too packed so that'll be likely something to look into in the future.

**Reply by shivan-s** [reply]
That's right mate.

https://github.com/nutrify/

Hopefully this is you? Must be from the icon.

**Comment by mrdbourke** [comment]
Not me 🤔  - someone must have made it hahaha

But looks exactly like what we're doing

**Comment by mrdbourke** [comment]
Seems they've got no public members, I wonder how we can contact

**Comment by shivan-s** [comment]
Haha damn.

Tough pill to swallow but some other names to consider: nutrifi, nutrifAI,
nutrif-eye, nutrAI.

**Comment by mrdbourke** [comment]
Website is live: https://www.nutrify.app/

Will add more to it in coming weeks.

**Comment by shivan-s** [comment]
Well done Dan :)
